Rubberduck.Core/Refactorings/MoveCloserToUsage/MoveCloserToUsageRefactoring.cs

```diff
@@ -168,9 +169,12 @@ private void InsertNewDeclaration()
             }
 
             var insertionIndex = (expression as ParserRuleContext).Start.TokenIndex;
+            var firstReferenceLine = _vbe.ActiveCodePane.CodeModule.GetLines((expression as ParserRuleContext).Start.Line, 1);
+            var indentLength = firstReferenceLine.Length - firstReferenceLine.TrimStart().Length;
+            var padding = new string(' ', indentLength);
 
             var rewriter = _state.GetRewriter(firstReference.QualifiedModuleName);
-            rewriter.InsertBefore(insertionIndex, newVariable);
+            rewriter.InsertBefore(insertionIndex, newVariable + padding);
```

Comment:
I wonder if given the fact that we create a `newVariable` which ends with a newline, it means that we can just look at the `expression`'s `Column` property and pad that many? That would avoid the COM access altogether.